example/spec/emsReport/emsReportSpec.js

```diff
@@ -0,0 +1,71 @@
+const moment = require('moment');
+const { Lambda } = require('aws-sdk');
+const { loadConfig } = require('../helpers/testUtils');
+const aws = require('@cumulus/common/aws');
+
+const config = loadConfig();
+
+const emsReportLambda = `${config.stackName}-EmsReport`;
+const bucket = config.bucket;
+const emsProvider = config.ems_provider;
+const stackName = config.stackName;
+
+describe('The EMS report', () => {
+  describe('The EmsReport lambda is invoked daily', () => {
+    let expectReports = false;
+    beforeAll(async () => {
+      const lambda = new Lambda();
+      const lambdaConfig = await lambda.getFunctionConfiguration({ FunctionName: emsReportLambda })
+        .promise();
+      const lastUpdate = lambdaConfig.LastModified;
+
+      // Compare lambda function's lastUpate with the time 24 hours before now.
+      // If the lambda is created 24 hours ago, it must have been invoked
+      // and generated EMS reports for the previous day.
+      if (new Date(lastUpdate).getTime() < moment.utc().subtract(24, 'hours').toDate().getTime()) {
+        expectReports = true;
+      }
+    });
+
+    it('reports are generated', async () => {
```

Comment:
For consistency, imagine that `it` as well as the previous `describe` statements are part of a sentence you are creating to describe the behavior.  This is the way it ends up getting printed in the test reports

What you have reads:  "The EMS report: The EmsReport lambda is invoked daily (it) reports are generated"

I'd suggest changing the previous `describe` to "When run automatically" and the `it` statement to "generates an EMS report every 24 hours".

The sentence then reads: "The EMS report: When run automatically (it) generates an EMS report every 24 hours"

Reply:
updated the wordings for this test.